README.md

```diff
@@ -56,7 +56,7 @@ $ yarn add react-router-dom
 
 $ rails g resource Crawl title:string user_id:integer
 $ rails g resource Bar yelp_id:string name:string image-url:string rating:float adress1:string address2:string address3:string city:string zip_code:integer country:string state:string display_address:string phone:string display_phone:string
-$ rails g resource JoinTable bar_id:integer crawl_id:integer
+$ rails g resource JoinTable bar_id:references crawl_id:references
```

Comment:
Looks like this is sorta notes for the team which is fine. Just know that usually README files are for displaying info about serving the project and documentation.

Reply:
I suggest using a different name for the `JoinTable` resource. Perhaps look at naming conventions for join tables.